report_thesis/src/sections/background.tex

```diff
@@ -226,28 +226,40 @@ \subsubsection{Extra Trees Regressor (ETR)}
 However, it often achieves better generalization performance, especially in high-dimensional or noisy datasets.
 
 \subsubsection{Gradient Boosting Regression (GBR)}\label{sec:gradientboost}
-In this section we introduce \gls{gbr} primarily based on \citet{James2023AnIS}.
-\gls{gbr} is an ensemble learning method that sequentially construct multiple decision trees, each one correcting the errors of the previous one, using gradient descent and boosting techniques.
-Each tree is small, with few terminal nodes, preventing large adjustments based on a single tree's predictions.
-This ensures that each tree makes small and simple error corrections, such that each step refines the model's performance more reliably.
+Continuing with the discussion of ensemble learning models, we introduce \gls{gbr} based on Chapter 10.10 of \citet{hastie_elements}.
```

Comment:
I might be wrong, but isnt it typical to include the relevant pages in the cite? Meaning, we add these pages in the references, rather than specifying them in the text?